CONTRIBUTING.md

```diff
@@ -26,6 +27,100 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Hack On Libpod
```

Comment:
I'm not a fan of the word "Hack".  Could we change it to "Contributing to Libpod"?

Reply:
Sure